cmd/main.go

```diff
@@ -115,14 +118,14 @@ func buildCmdTree() *cobra.Command {
 	// kubebuilder init
 	rootCmd.AddCommand(newInitCmd())
 
-	// kubebuilder update (v1 only)
-	if internal.ConfiguredAndV1() {
-		rootCmd.AddCommand(newUpdateCmd())
-	}
-
 	// kubebuilder version
 	rootCmd.AddCommand(version.NewVersionCmd())
 
+	// kubebuilder update (v1 only) to allow update the project
+	if !internal.IsVersionSupported() {
+		rootCmd.AddCommand(newUpdateCmd())
+	}
+
```

Comment:
Okey, this was partly my fault. I know I said last, but I forgot about `NewVersionCmd`. It should be just before `NewVersionCmd`.

Reply:
Tks. Make sense done.